rclip/utils/helpers.py

```diff
@@ -151,6 +151,20 @@ def init_arg_parser() -> argparse.ArgumentParser:
     default=400,
     help="preview height in pixels; default: 400",
   )
+  parser.add_argument(
+    "--model-name",
+    "-M",
+    type=str,
+    default="ViT-B-32",
+    help="OpenCLIP model name",
+  )
+  parser.add_argument(
+    "--checkpoint-name",
+    "-C",
+    type=str,
+    default="datacomp_xl_s13b_b90k",
```

Comment:
The change to a better model sounds exciting, but how compatible is this model with the `ViT-B-32-quickgelu`? What will happen to the existing users who will upgrade to the new `rclip` version once we merge this PR?

If the new model is not compatible with `ViT-B-32-quickgelu`, which it probably is, we should:
- warn the user that we are going to reindex their images after the upgrade
- reindex all already indexed images after the upgrade

Another option probably, better:
- warn the user that the default model has changed
- let them opt out of the change and keep using the older one

What do you think?